packages/react/src/components/molecules/FooterLinks/index.js

```diff
@@ -9,30 +9,30 @@ import PropTypes from 'prop-types';
 import SidebarHeading from 'MayflowerReactHeadings/SidebarHeading';
 
 const FooterLinks = (footerLinks) => (
-  <section className="ma__footer-links">
+  <nav className="ma__footer-links" aria-label="Footer navigation">
     {
       footerLinks.items.map((footerLinksNav, i) => (
         /* eslint-disable-next-line react/no-array-index-key */
         <FooterLinksNav {...footerLinksNav} showNavHeading={footerLinks.showNavHeading} key={`footerLinksNav_${i}`} index={i} />
       ))
     }
-  </section>
+  </nav>
 );
 
 const FooterLinksNav = (footerLinksNav) => (
-  <nav aria-labelledby={footerLinksNav.id} className={`ma__footer-links__nav ${footerLinksNav.showNavHeading ? ' ma__footer-links__nav--heading' : ''}`}>
+  <div className={`ma__footer-links__nav ${footerLinksNav.showNavHeading ? ' ma__footer-links__nav--heading' : ''}`}>
     {
-      footerLinksNav.showNavHeading ? <SidebarHeading title={footerLinksNav.heading} level={2} /> : <h2 className="visually-hidden" id={footerLinksNav.id}>{footerLinksNav.heading}</h2>
+      footerLinksNav.showNavHeading ? <SidebarHeading title={footerLinksNav.heading} level={2} /> : <h2 className="visually-hidden">{footerLinksNav.heading}</h2>
```

Comment:
`id` is a required prop, I'm not sure why it was required, but I don't think we should remove it until we can make sure it's not used and remove it from the proptype check

Reply:
ah ok, maybe it's required for the aria label. I will remove it from data